doc/WebSite/EF-Core-Sqlite-Integration.md

```diff
@@ -0,0 +1,71 @@
+### Download Starter Template
+
+Download a starter template with **ASP.NET Core** and **Entity Framework Core** to integrate SqlLite. 
+[Multi-page template with **ASP.NET Core 2.x** + **.Net Core Framework** + **Authentication**](https://aspnetboilerplate.com/Templates) 
+will be explained in this document.
+
+### Install 
+
+Install [`Microsoft.EntityFrameworkCore.Sqlite`](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.Sqlite/) nuget package to **\*.EntityFrameworkCore** project. 
+
+### Configuration
+
+Some configurations and workarounds are needed to use Sqlite with ASP.NET Core and Entity Framework Core. 
+
+#### Configure DbContext 
+
+Replace `YourProjectNameDbContextConfigurer.cs` with the following lines
+
+```c#
+public static class SqliteDemoDbContextConfigurer
+{
+    public static void Configure(DbContextOptionsBuilder<SqliteDemoDbContext> builder, string connectionString)
+    {
+        builder.UseSqlite(connectionString);
+    }
+
+    public static void Configure(DbContextOptionsBuilder<SqliteDemoDbContext> builder, DbConnection connection)
+    {
+        builder.UseSqlite(connection);
+    }
+ }
+ ```
+
+#### Configure connection string 
+
+Change connection string to your Sqlite connection in ***.Web.Mvc/appsettings.json**. For example:
+
+```js
+{
+  "ConnectionStrings": {
+    "Default": "Data Source=SqliteDemoDb"
+  },
+  ...
+}
+
+```
+
+#### A workaround
+
+To prevent EF Core from calling `Program.BuildWebHost()` rename `BuildWebHost`. For example, change it to `InitWebHost`. 
+This workaround won't be detailed in this document, but you will find some points to understand better. Check the following issues,
```

Comment:
To understand why it needs to be renamed check the following issues,

Reply:
"To understand why it needs to be renamed`,` check the following issues`:`"